### DISCUSSION TITLE
Using VRP from google-or tools in my c++ cmake project

### DISCUSSION THREAD

**Discussion by komalbb** [google/or-tools#2577] [Build and configuration issues] [answered]
Hi,

I am not able to find a way on how to include only the vrp module from google or tools in my cmake project.

I have build or-tools from source and then did make install in /usr/local/.

But I need help on steps on how to install google or-tools in c++ make project.

Did this:
wget https://github.com/google/or-tools/releases/download/v9.0/or-tools_ubuntu-18.04_v9.0.9048.tar.gz 
tar -zxvf or-tools_ubuntu-18.04_v9.0.9048.tar.gz 
then copy the files under lib/or-tools
set(CMAKE_PREFIX_PATH ${CMAKE_SOURCE_DIR}/lib)

But it seems like it's not working; I have or-tools also installed globally at /usr/local. how to uninstall it at /usr/local and make the dependency works.

Thanks for your help

**Comment by lperron** [comment]
You can have a look at: https://github.com/or-tools/cmake_or-tools

**Comment by Mizux** [comment, marked as answer]
Few points:

* Artifacts are generated using the legacy Makefile -> they won't contain any CMake releated stuff, sorry
* Currently we don't have uninstall rules and install rules is heavily based on copy file using globing pattern
* If using the Makefile or CMake (with Makefile generator) based build, we do support the `DESTDIR` env to install in a directory.
https://www.gnu.org/prep/standards/html_node/DESTDIR.html

* If CMake you can also use https://cmake.org/cmake/help/latest/variable/CMAKE_INSTALL_PREFIX.html
